File: api/src/main/java/com/facundocastro/api/model/Educacion.java
package com.facundocastro.api.model;

import lombok.Data;

import javax.persistence.*;
import java.util.Date;

@Data
@Entity
public class Educacion {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String institucion;
    private String descripcion;
    @Temporal(TemporalType.DATE)
    private Date fechaI;
    @Temporal(TemporalType.DATE)
    private Date fechaF;
}

File: angular-portfolio/src/app/models/Educacion.ts
class Educacion {
    protected _id: number = -1;
    protected _institucion: string = '';
    protected _descripcion: string = '';
    protected _fechaI: Date = new Date();
    protected _fechaF: Date = new Date();

    public constructor() {}

    public get id(): number {
        return this._id;
    }
    public set id(value: number) {
        this._id = value;
    }

    public get institucion(): string {
        return this._institucion;
    }
    public set institucion(value: string) {
        this._institucion = value;
    }

    public get descripcion(): string {
        return this._descripcion;
    }
    public set descripcion(value: string) {
        this._descripcion = value;
    }

    public get fechaI(): Date {
        return this._fechaI;
    }
    public set fechaI(value: Date) {
        this._fechaI = value;
    }

    public get fechaF(): Date {
        return this._fechaF;
    }
    public set fechaF(value: Date) {
        this._fechaF = value;
    }
}